src/NuGet.Services.BasicSearch/Startup.cs

```diff
@@ -297,9 +294,6 @@ private void TrackIndexMetrics(NuGetSearcherManager searcherManager, SearchTelem
                         case "/search/query":
                             await ServiceEndpoints.V2SearchAsync(context, _searcherManager, _responseWriter);
                             break;
-                        case "/rankings":
```

Comment:
I feel like removing these endpoints is a little bit out of the scope of what a "SemVer2 filtering" PR should do, but I'm fine with it being in here.